ha/core/event_analyzer/parser/parser.py

```diff
@@ -188,9 +198,18 @@ def parse_event(self, msg: str) -> HealthEvent:
             event_type = cluster_resource_alert[EventAttr.EVENT_PAYLOAD.value][HealthAttr.RESOURCE_STATUS.value]
             specific_info = cluster_resource_alert[EventAttr.EVENT_PAYLOAD.value][HealthAttr.SPECIFIC_INFO.value]
             if resource_type == CLUSTER_ELEMENTS.NODE.value:
+                node_type = None
+                if node_id in self.data_node_ids:
```

Comment:
Can we get the pod-type from k8s monitor itself ? instead of fetching type from cluster.conf file.
K8s monitor return too many fields just like genartion_id, we can fetch pod-type too. which will reduce unnecessary overload of this iteration to check pod-type. In pod description we've 2 fields which talk about pod-type 
Last 2 fields in Labels
```
[root@ssc-vm-g4-rhev4-0314 ~]# kubectl describe pod cortx-data-ssc-vm-g4-rhev4-0762-68dfc746cd-ngr8c
Name:         cortx-data-ssc-vm-g4-rhev4-0762-68dfc746cd-ngr8c
Namespace:    default
Priority:     0
Node:         ssc-vm-g4-rhev4-0762.colo.seagate.com/10.230.250.84
Start Time:   Thu, 14 Apr 2022 02:48:58 -0600
Labels:       app=cortx-data-ssc-vm-g4-rhev4-0762
              cortx.io/hax-enabled=true
              cortx.io/machine-id=113037cdcd2443bda5b6b47acd312421
              cortx.io/service-type=cortx-data
              name=cortx-data
```

Reply:
as per discussion get type list from conf will keep it @akash2144